backend-elasticsearch/pom.xml

```diff
@@ -15,6 +15,9 @@
     </parent>
     <artifactId>hibernate-search-backend-elasticsearch</artifactId>
 
+    <name>Hibernate Search Backend - Elasticsearch</name>
+    <description>Hibernate Search Backend relying on remote Elasticsearch clusters</description>
```

Comment:
Ah cool, that's been bugging me for a while. I thought we would fix it when merging but that's good to have it fixed now.